qoffer-infrastructure/src/main/groovy/life/qbic/Presenter.groovy

```diff
@@ -0,0 +1,21 @@
+package life.qbic
+
+import life.qbic.portal.portlet.customers.create.CreateCustomerOutput
+import life.qbic.portal.portlet.customers.update.UpdateCustomerOutput
+import life.qbic.portal.portlet.offers.create.CreateOfferOutput
+import life.qbic.portal.portlet.offers.search.SearchOffersOutput
+import life.qbic.portal.portlet.offers.update.UpdateOfferOutput
+
+/**
+ *  Handles the presentation of the qoffer-2.0 use cases and its internal data model to the viewmodel.
+ *
+ *  This class acts as connector between the use cases of qoffer-2.0 and PortletView
+ *  and should be used to transfer data between these components.
+ *  It converts the use case output into data that is stored in the {@link PortletView}.
```

Comment:
```suggestion
 *  It converts the use case output into data that is stored in the {@link ViewModel}.
```

Reply:
Wait, i thought PortletView is intended to be the viewModel?

Reply:
nope. The portlet view is the vaadin stuff. the data for the portlet view is stored elsewhere. mutliple views could have access to the same data and therefore manipulating the viewmodel suffices to update the views. The responsibility of keeping the representation of the data up to date lies with the view but the data resides in the viewmodel.

Reply:
Thanks for clearing that up for me!